scripts/system/create/entityProperties/html/js/entityProperties.js

```diff
@@ -2856,7 +2890,7 @@ function createTextureProperty(property, elProperty) {
 
     let imageLoad = function(url) {
         elDiv.style.display = null;
-        if (url.slice(0, 5).toLowerCase() === "atp:/") {
+        if (url.slice(0, 5).toLowerCase() === "atp:/" || url.slice(0, 6).toLowerCase() === "file:/") {
```

Comment:
Is this change needed still?

Reply:
Yes. This is possible to use such "file://" URL (it works)
despite the fact that it's not something we would like to do promotion of.
So the fix is still relevant, to not have a broken preview.
Such relative URLs can't be resolved bu the web component (as for "atp:/") so this code is to address that.